app/scripts/controllers/rif/rns/register/index.js

```diff
@@ -180,4 +182,32 @@ export default class RnsRegister extends RnsJsDelegate {
     return `${dataSignature}${dataOwner}${dataSecret}${dataDuration}${dataName}`;
   }
 
+  /**
+   * Overrides the parent operation because rns-js is failing on this, we invoke the contract directly.
+   */
+  createSubdomain (domainName, subdomain, ownerAddress, parentOwnerAddress) {
+    domainName = this.addRskSuffix(domainName);
+    if (!ownerAddress) {
+      ownerAddress = this.address;
+    }
+    const node = namehash(domainName);
+    const label = web3Utils.sha3(subdomain);
+    const result = this.send(this.rnsContractInstance, 'setSubnodeOwner', [node, label, ownerAddress])
+    result.then(transactionReceipt => {
+      if (transactionReceipt) {
```

Comment:
If `transactionReceipt` is not null or undefined doesnt mean that the transaction was succesful. At least if nifty isnt overriding the default behaviour of web3 with some kind of middleware.

We need to check the transctionReceipt.status to check if it is true or false

This is in general, on every transaction

Reply:
Noted! can we create a task to do this and work on that in another story creating a new branch for it. I think it could be better to separate that.

Reply:
Done now we have the transaction listener to manage this.